flink-runtime/src/main/java/org/apache/flink/runtime/rest/handler/AbstractHandler.java

```diff
@@ -268,12 +266,7 @@ private CompletableFuture<Void> handleException(
                     HttpResponseStatus.SERVICE_UNAVAILABLE,
                     responseHeaders);
         } else {
-            if (throwable instanceof UnavailableDispatcherOperationException
-                    || throwable instanceof FileNotFoundException) {
-                log.debug("Job is not initialized or is finished: {}", throwable.getMessage());
-            } else {
-                log.error("Unhandled exception.", throwable);
-            }
+            log.error("Unhandled exception.", throwable);
```

Comment:
I don't understand this change.
If you want an e2e test to ignore a certain exceptions in the logs, then ignore that exception in `common.sh`.